ethereum/spec/src/test/java/tech/pegasys/teku/spec/SpecFactoryTest.java

```diff
@@ -46,7 +46,9 @@ public void defaultFactoryShouldNotEnableBellatrixUnlessForkEpochIsSet(final Str
     final Spec spec = SpecFactory.create(configName);
     if (configName.equals("minimal")) { // Minimal doesn't have altair scheduled
       assertThat(spec.getForkSchedule().getSupportedMilestones()).containsExactly(PHASE0);
-    } else if (configName.equals("ropsten") || configName.equals("kiln")) {
+    } else if (configName.equals("ropsten")
+        || configName.equals("kiln")
+        || configName.equals("sepolia")) {
```

Comment:
nit: Might be time to add a static `Set` with the list of bellatrix enabled testnets rather than individually listing them here (then do a .contains here)